src/db/index.ts

```diff
@@ -1,5 +1,5 @@
 import { ViolationInfo } from "../lint";
-import { CommitUser } from "types/github";
+import { CommitUser } from "~/types/github";
```

Comment:
IIRC typescript does not translate the import path according `paths` definitions, so at runtime it'll tries to import `~types/github` and fail. This works before since we're importing types only and these imports are elided from output. Sorry for not noticing this before, but I think it's better to use a regular relative import when we are not using any bundling tools.